packages/coinstac-client-core/src/index.js

```diff
@@ -89,7 +89,7 @@ class CoinstacClient {
     this.appDirectory = opts.appDirectory ||
       CoinstacClient.getDefaultAppDirectory();
     this.halfpennyBaseUrl = opts.hp;
-    this.logger = opts.logger || new Logger({ transports: [new Console()] });
+    this.logger = opts.mainLogger || new Logger({ transports: [new Console()] });
```

Comment:
We should keep this as `logger` so it’s more general. _coinstac-client-core_ should be environment-agnostic – unaware that it's running in a “main” Electron process.